### PR TITLE
Gallery overlay view does not cover whole screen when presented in a modal view controller

### PR DESCRIPTION
**Pull request by IvoPintodaSilva**
Overlay view added to the back of the gallery instead of the original view controller.

### CODE REVIEW

File: ImageViewer/Source/GalleryViewController.swift [hunk 1]
```diff
@@ -149,16 +149,17 @@ open class GalleryViewController: UIPageViewController, ItemControllerDelegate {
 
         NotificationCenter.default.removeObserver(self)
     }
-
+
```
Comment (zfoltin) [inline ImageViewer/Source/GalleryViewController.swift]: Unnecessary whitespace

File: ImageViewer/Source/GalleryViewController.swift [hunk 2]
```diff
@@ -149,16 +149,17 @@ open class GalleryViewController: UIPageViewController, ItemControllerDelegate {
...
-
-        if let controller = self.presentingViewController {
-
-            overlayView.center = controller.view.boundsCenter
-            controller.view.addSubview(overlayView)
-        }
+        overlayView.center = CGPoint(x: (UIScreen.main.bounds.width / 2), y: (UIScreen.main.bounds.height / 2))
+        
+        /// Add the overlay to the back of the gallery view controller instead of the original one
+        /// This fixes the problem of not opening the gallery in fullScreen when it's originated on a Modal controller
```
Comment (zfoltin) [inline ImageViewer/Source/GalleryViewController.swift]: Let's get rid of these comments, they are meaningless without knowing how it worked before

P.S. we generally comment using `//` not `///`

File: ImageViewer/Source/GalleryViewController.swift [hunk 3]
```diff
@@ -149,16 +149,17 @@ open class GalleryViewController: UIPageViewController, ItemControllerDelegate {
 
         NotificationCenter.default.removeObserver(self)
     }
-
+    
     fileprivate func configureOverlayView() {
 
+        /// Make the overlay view occupy the whole screen
```
Comment (zfoltin) [inline ImageViewer/Source/GalleryViewController.swift]: I don't think that's what the next line does. The intention here is to give the overlay enough padding so it still covers the screen when rotated.
Let's get rid of this comment, people will understand what the next line does without it

**Comment by IvoPintodaSilva** [comment]
Comments resolved, @zfoltin

However, at first, I didn't understand why the overlayView size wasn't simply "UIScreen.main.bounds.size".

Maybe because in my use case I only use the gallery in portrait mode, I didn't account for the rotation. Maybe a comment on that would turn it clearer for future contributors. 
Just a suggestion, keep up the amazing work 💪
